### ISSUE TITLE
Item name display changes

### ISSUE THREAD

**Issue by xanthian** [Micalobia/Full-Slabs#55]
See screenshot 
![image](https://user-images.githubusercontent.com/7688001/138741176-d3c40252-7246-46c6-be94-cb18b25b3e68.png)

This is a glowstone slab from Consistency+ (which I may add works great with your mod)
However the name display in WTHIT is showing Full Slab rather than the correct name tag.

Not really a huge issue, but strange nonetheless as some do display correctly like this 1, right next to the one above.
![image](https://user-images.githubusercontent.com/7688001/138741713-62bd6a44-502b-44a9-9498-43c3f29dc2d3.png)

ps..
not an issue with Consistency+ tags, as when not used in conjunction with another slab it displays the correct name
![image](https://user-images.githubusercontent.com/7688001/138741872-55c0bbc1-a8f3-420f-a336-c23cf2db61df.png)

**Comment by Micalobia** [comment]
Yeah, it's something that happens when slabs are mixed because that creates a different type of block, might need to look into  more direct WTHIT compatibility
